src/views/AddItem.jsx

```diff
@@ -62,71 +66,115 @@ export function AddItem({ data, listToken }) {
 
 				// put a note that the form was submitted (and erase after 5 seconds)
 				setSubmissionConfirmation(
-					`${itemName} was added to your shopping list.`,
+					`${
+						itemName[0].toUpperCase() + itemName.slice(1)
+					} was added to your shopping list.`,
 				);
 				setTimeout(setSubmissionConfirmation, 5000);
 			})
 			.catch((error) => {
 				setSubmissionConfirmation(
-					`There was a problem adding your item, please try again.`,
+					`There was a problem adding your item, please try again later.`,
 				);
 				console.log(error);
 			});
 	};
 
 	return (
 		<>
-			<form onSubmit={submitForm}>
-				<label htmlFor="itemName">
-					Item Name
+			<div className="bg-main text-white w-full h-[80%] absolute bottom-0 max-w-md animate-openAddItem overflow-clip z-10">
+				<div className="w-full absolute bottom-0 animate-appear">
+					<AddItemBackground className="w-full h-full fill-main-darkest" />
+				</div>
+				<form
+					onSubmit={submitForm}
+					className="grid justify-items-center w-full mt-10 relative"
+				>
+					<h1 className="font-bold uppercase text-center my-5">add new item</h1>
```

Comment:
Not a fan of the font size. Since this is an `h1`, it's the most important heading of the AddItem view and should look it as well. I also added some letter spacing since it looks too condensed to me.
```suggestion
					<h1 className="text-2xl tracking-wide font-bold uppercase text-center my-5">add new item</h1>
```